src/envoy/http/jwt_auth/jwt_authenticator_test.cc

```diff
@@ -472,7 +472,8 @@ TEST_F(JwtAuthenticatorTest, TestMissedJWT) {
 }
 
 TEST_F(JwtAuthenticatorTest, TestAllowMissingOrFailedIsTrue) {
-  // When JWT is missing from the config, the status should be OK
+  // In this test, when JWT is missing, the status should still be OK
```

Comment:
The test case you want is when jwt spec is available, and request contain invalid jwt, but request will be accepted. This test is ok to add, but not for that purpose

Reply:
Done: added another test when the request contains an invalid JWT but still returns OK because allow_missing_or_failed is true.